mod/01_rating/set/type/metric_value.rb

```diff
@@ -20,13 +20,17 @@ def company_card
   Card.fetch company_name
 end
 
+def source_subcards new_source_card
+  [new_source_card.subfield :file, new_source_card.subfield :text,
+    new_source_card.subfield :wikirate_link]
+end
+
 def source_exist?
   sub_source_card = subfield('source')
-  return false if sub_source_card.nil?
+  return false if sub_source_card.nil? ||
+                  sub_source_card.subcard('new_source').nil?
```

Comment:
doesn't this mean sources are no longer required?

(I didn't really intend to merge this yet.  doh.)